dcos-log/api/v2/handlers.go

```diff
@@ -0,0 +1,293 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/dcos/dcos-go/dcos"
+	"github.com/dcos/dcos-go/dcos/nodeutil"
+	"github.com/dcos/dcos-log/dcos-log/api/middleware"
+	"github.com/dcos/dcos-log/dcos-log/mesos/files/reader"
+	"github.com/gorilla/mux"
+)
+
+const (
+	prefix = "/system/v1/agent"
+)
+
+// ERROR is a http.Error wrapper, that also emits an error to a console
+func ERROR(w http.ResponseWriter, msg string, code int) {
```

Comment:
Any reason this is in all caps vs just `Error` ?  Also, does it need to be exported?  After reading a little bit it seems like you're trying to indicate it's a macro of sorts.  Seems weird for golang code though.

Reply:
Also, if you pass the request to ERROR, you could log more context about the request that caused the error (url, query parameters, any relevant headers, and so on)

Reply:
i'll leave it as is for now, we will make more improvements after beta2.
I will leave TODO comment